### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by edmondsylar**

### CODE REVIEW

**Comment by SokoPaulSokool** [comment]
its very important to add [(Finishes|Fixes|Delivers) #TRACKER_STORY_ID] 
eg **add user page with order [Finishes #160414064]**
pivotal tracker  will change the specified story to the given state

**Review by Ivankivu — COMMENTED** [review]
fair enough. I can see u made corrections, please here it okay

File: UI/home.html [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+    <meta charset="utf-8">
+    <title></title>
+    <link rel="stylesheet" href="css/home2.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css" integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU" crossorigin="anonymous">
+  </head>
+  <body>
+    <header>
+      <div class="navbar">
+        <a href="#home" class="active"><i class="fa fa-home"></i> Home</a>
+        <a href="#add_item"><text> <i class="fa fa-fort-awesome-alt"> Fast-Food-Fast - Admin Panel</i></text></a>
```
Comment (Ivankivu) [inline UI/home.html]: I do see a tag 'text', please confirm if it's meant to be here. Otherwise if that gets sort, then you can merge

**Comment by sekayasin** [comment]
@edmondsylar u've not provided a brief description on this PR so that we can quickly get an insight on what u've pushed and also provide a link to your hosted Github pages in the READMe.. #happyCoding

**Comment by kdaglas** [comment]
was about to say the same that @sekayasin  said... and another thing your commit messages have to have one tense, either use 'Add' or use 'Added' but not both... otherwise the rest is cool...

**Comment by 66hakeem** [comment]
Your javascript could be kept in an external file which can be imported in the script tags.

**Comment by mozzy22** [comment]
@edmondsylar  its cool work. can u place all the java script outside in a separate js file?

**Comment by edmondsylar** [comment]
Tho I have been changing the statuses manually but this seems to be a good
practice am going to work on adopting this instead
Thank You very much

**Pull request merged**